.travis.yml

```diff
@@ -8,68 +8,47 @@ php:
 
 env:
   - DRUPAL_CORE=8.1.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
   - DRUPAL_CORE=8.2.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
+  - REVIEW_STANDARD="Drupal"
+    CODE_REVIEW=1
+  - REVIEW_STANDARD="DrupalPractice"
+    CODE_REVIEW=1
+
+matrix:
+  include:
+    - php: 5.5
+      env: CODE_REVIEW=1
```

Comment:
It would be nicer to run the coding standards check on the fastest PHP version, instead of on the slowest one :)